config/shared.lua

```diff
@@ -1,168 +1,168 @@
 return {
     locations = {
-        ['main'] = {
+        main = {
             label = 'Truck Shed',
-            coords = vector3(153.68, -3211.88, 5.91),
-            size = vector3(3.0, 3.0, 5.0),
+            coords = vec3(153.68, -3211.88, 5.91),
+            size = vec3(3.0, 3.0, 5.0),
             rotation = 274.5,
             icon = 'fa-solid fa-eye',
             debug = false
         },
-        ['vehicle'] = {
+        vehicle = {
             label = 'Truck Storage',
-            coords = vector3(141.12, -3204.31, 5.85),
-            size = vector3(5.0, 5.0, 5.0),
+            coords = vec3(141.12, -3204.31, 5.85),
+            size = vec3(5.0, 5.0, 5.0),
             rotation = 267.5,
             debug = false
         },
-        ['stores'] = {
+        stores = {
             [1] = {
```

Comment:
When we change stuff in the config already then what about getting rid of this god awfull [1] index.

same reason we did it inside the jobs table just no reason to index the table by the number it will have by default when we just pass it as a normal array.